src/group_theory/group_action/defs.lean

```diff
@@ -52,6 +52,9 @@ class has_vadd (G : Type*) (P : Type*) := (vadd : G → P → P)
 @[to_additive has_vadd]
 class has_scalar (M : Type*) (α : Type*) := (smul : M → α → α)
 
+@[to_additive]
+instance has_mul.to_has_scalar (α : Type*) [has_mul α] : has_scalar α α := ⟨(*)⟩
```

Comment:
There's a `smul_eq_mul` simp lemma somewhere - can you move that to nearer here and generalize it

Reply:
It would be nice to reference `monoid.to_mul_action` and `mul_zero_class.to_smul_with_zero` in a docstring here, as we do with monoid.to_mul_action.